src/allmydata/test/common.py

```diff
@@ -94,9 +94,9 @@
 TEST_RSA_KEY_SIZE = 522
 
 EMPTY_CLIENT_CONFIG = config_from_string(
```

Comment:
Just FYI, I encountered this in one of my branches, which I guess I should put on hold until you're done with node.py, and decided it was easier if configs were unicode. But that might be better as later iteration, so if this is minimal change to keep going, that's fine.